src/main/java/HumanPlayer.java

```diff
@@ -1,14 +1,14 @@
-public class HumanTurn {
+public class HumanPlayer implements Player {
 
-    private final Board board;
-    private final UserInterface gameUI;
+    private Board board;
+    private UserInterface gameUI;
 
-    public HumanTurn(Board board, UserInterface ui) {
-        this.board  = board;
+    public HumanPlayer(UserInterface ui, String mark) {
         this.gameUI = ui;
     }
 
-    public int getCellIndex() {
+    public int getCellIndex(Board board) {
+        this.board = board;
```

Comment:
Is the board used in any other method of HumanPlayer? If not, maybe it can be a local variable?